automol/form/_form.py

```diff
@@ -287,13 +267,12 @@ def _sort_key(entry):
     )
 
 
-def sort_vector(fml, symbs=None):
-    """Generate a sort vector for sorting various formulas against each other
+def sort_vector(fml: str, symbs: list[str] | None = None):
+    """Generate a sort vector for sorting various formulas against each other.
 
     :param fml_str: stochiometric chemical formula string
-    :type fml_str: str
     :param symbs: atomic symbols in the desired sort order (optional)
-    :type symbs: list[str]
+    :return: Sorted vector
```

Comment:
Should be `:return: The sort vector` (a vector value used for sorting formulas in a sequence, based on their atom counts).